### ISSUE TITLE
popUntil method definition is wrong

### ISSUE THREAD

**Issue by federicoviceconti** [yagoliveira92/seafarer#4]
I think this method is wrong:

`void popUntil(void Function(Route<dynamic>) predicate)`

The parameter should be: `bool Function(Route<dynamic>)` 

Inside the method there was made an unnecessary cast:
```
void popUntil(void Function(Route<dynamic>) predicate) {
    this
        .navigatorKey!
        .currentState!
        .popUntil(predicate as bool Function(Route<dynamic>));
}
```

**Comment by daniloapr** [comment]
I'm having problem with popUntil too.

**Comment by daniloapr** [comment]
I fixed locally and it worked:

``` dart
  void popUntil(bool Function(Route<dynamic>) predicate) {
    this
        .navigatorKey!
        .currentState!
        .popUntil(predicate);
  }
```

How ever, casting to not null with `!` is not the best solution.

**Comment by yagoliveira92** [comment]
Fix in the release 0.1.1, thanks @daniloapr for contribution, and sorry for the publication to late.
